### PR TITLE
Fixes #8619 - Add AMSI docs

### LINKED ISSUE #8619: Add AMSI method invocation logging as experimental feature
### Prerequisites

- [X] Write a descriptive title.
- [X] Search the existing issues.

### Summary of the new feature or changed behavior

Name       : PSAMSIMethodInvocationLogging
Description: Provides Antimalware Scan Interface (AMSI) notification of .NET method invocations.

### List of articles that need to be updated

https://docs.microsoft.com/en-us/powershell/scripting/learn/experimental-features?view=powershell-7.3

### Link to related PR in PowerShell/PowerShell repo

https://github.com/PowerShell/PowerShell/pull/16496

### Link to related Issues in PowerShell/PowerShell repo

n/a

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fixes AB#1927076 - Fixes #8619 - Add AMSI docs

## PR Context

Check the boxes below to indicate the content affected by this PR.

**Repository or docset configuration**
- [ ] Repo documentation and configuration (.git/.github/.vscode etc.)
- [ ] Docs build files (.openpublishing.* and build scripts)
- [ ] Docset configuration (docfx.json, mapping, bread, module folder)

**Conceptual documentation**
- [x] Files in docs-conceptual

**Cmdlet reference & about_ topics**
When changing **cmdlet reference** or **about_ topics**, the changes should be copied to all
relevant versions. Check the boxes below to indicate the versions affected by this change.

- [ ] Preview content
- [ ] Version 7.2 content
- [ ] Version 7.1 content
- [ ] Version 7.0 content
- [ ] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**. If the PR is work in progress,
  please add the prefix `WIP:` or `[WIP]` to the beginning of the title and remove the prefix when
  the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/docs-conceptual/learn/experimental-features.md [hunk 1]
```diff
@@ -88,6 +89,19 @@ In this example, a job is started and a breakpoint is set to break when the `Set
 run. The runspace is stored in a variable and passed to the `Get-PSBreakPoint` command with the
 **Runspace** parameter. You can then inspect the breakpoint in the `$breakpoint` variable.
 
+## PSAMSIMethodInvocationLogging
+
+The Windows Antimalware Scan Interface (AMSI) is an API that allows application actions to be passed
+to an antimalware scanner, such as Windows Defender, to be scanned for malicious payloads. Beginning
```
Comment (michaeltlombardi) [inline reference/docs-conceptual/learn/experimental-features.md:95]:
```suggestion
The Windows Antimalware Scan Interface (AMSI) is an API that allows applications to pass actions 
to an antimalware scanner, such as Windows Defender, for detecting malicious payloads. Beginning
```

Just a quick pass at removing some passive voice and clarifying value for this feature.

File: reference/docs-conceptual/learn/experimental-features.md [hunk 2]
```diff
@@ -88,6 +89,19 @@ In this example, a job is started and a breakpoint is set to break when the `Set
...
+with PowerShell 5.1, PowerShell running on Windows 10 (and higher) passes all script blocks to AMSI.
+
+This experimental feature extends the data that is sent to AMSI for inspection. With this feature
+enabled, PowerShell now includes any invocation of a .NET method member.
```
Comment (michaeltlombardi) [inline reference/docs-conceptual/learn/experimental-features.md:99]: Does it include _any_ invocation (i.e. opt-in) or _all_ invocations by default?

**Review by michaeltlombardi — COMMENTED** [review]
A small suggestion and a question. Otherwise, LGTM!

File: reference/docs-conceptual/learn/experimental-features.md (hunk 2 above)
Comment (sdwheeler) [reply reference/docs-conceptual/learn/experimental-features.md]: See updated changes

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**